src/tennisSimulator/Menu.java

```diff
@@ -0,0 +1,76 @@
+package tennisSimulator;
+
+import java.util.Scanner;
+
+public class Menu {
+
+    private static final PlayersDatabase playersDatabase = new PlayersDatabase();
+    private static final Scanner scanner = new Scanner(System.in);
+
+    public void openTennisSimulatorMenu() {
+        boolean quit = false;
+        System.out.println("WELCOME TO TENNIS SIMULATOR\n");
+        printMenu();
+        while (!quit) {
+            int choice = scanner.nextInt();
+            scanner.nextLine();
+            switch (choice) {
+                case 0:
+                    printMenu();
+                    break;
+                case 1:
+                    playersDatabase.print();
+                    break;
+                case 2:
+                    addPlayerToDatabase();
+                    break;
+                case 3:
+                    quit = true;
+                    System.out.println("Exiting simulator...");
+                    break;
+                default:
+                    System.out.println("Incorrect input. Please choose one of available options.\n");
+                    printMenu();
+                    break;
+            }
+        }
+    }
+
+    private void printMenu() {
+        System.out.println("Select:\n" +
+                "0 - show options;\n" +
+                "1 - show players database;\n" +
+                "2 - add player to database;\n" +
+                "3 - exit simulator;");
+    }
+
+    private void addPlayerToDatabase() {
+        System.out.print("Enter a full name: ");
+        String name = scanner.nextLine();
+        if (!name.contains(" ")) {
+            System.out.println("Addition failed. Please enter full player name");
+            return;
+        }
+
+        System.out.print("Enter a gender (man/woman): ");
+        String gender = scanner.nextLine().toLowerCase();
+
+        if (isValidGender(gender)) {
```

Comment:
Now naming is much better but I see several problems with this code.
isValidGender method checks whether gender is valid or not. Which is OK. But it also asks for user input. Imagine if someone adds some other method in this class where he needs to validate gender but he does not want user to enter anything. So this method definitely does too much and that "too much" is hidden behind naming. 
I would suggest you to have separate method for validating all fields (name, gender, rank), because I see that you already have validation, so you could group validation into single method and only if all are valid then create Player object. Other option is to have separate validate methods for each of the fields, but in any case you don't have to mix it with user input.
Also have a separate method for resolving the gender if you need to print different things depending on their gender.